docs/content/latest/secure/authentication/ldap-authentication-ycql.md

```diff
@@ -0,0 +1,235 @@
+   <td>Specifies the base directory to begin the user name search
+   </td>
+   <td>''
+   </td>
+  </tr>
+  <tr>
+   <td>ycql_ldap_bind_dn
+   </td>
+   <td>Specifies the username to perform the initial search when doing search + bind authentication
+   </td>
+   <td>''
+   </td>
+  </tr>
+  <tr>
+   <td>ycql_ldap_bind_passwd
+   </td>
+   <td>Password for username being used to perform the initial search when doing search + bind authentication
+   </td>
+   <td>''
+   </td>
+  </tr>
+  <tr>
+   <td>ycql_ldap_search_attribute
+   </td>
+   <td>Attribute to match against the username in the search when doing search + bind authentication. If no attribute is specified, the uid attribute is used
+   </td>
+   <td>''
+   </td>
+  </tr>
+  <tr>
+   <td>ycql_ldap_search_filter (not supported yet)
+   </td>
+   <td>The search filter to use when doing search + bind authentication
+   </td>
+   <td>''
+   </td>
+  </tr>
+</table>
+
+## Example
+
+To use LDAP password authentication on a new YugabyteDB cluster, follow these steps:
+
+1. Use tserver gflags to enable LDAP authentication on YB-TServer in simple bind mode. Use the below configuration to start a YugabyteDB cluster.
+
+    ```sh
+    --use_cassandra_authentication=true --ycql_use_ldap=true --ycql_ldap_server=ldap://ldap.forumsys.com:389 --ycql_ldap_user_prefix=uid= --ycql_ldap_user_suffix=, dc=example, dc=com --ycql_ldap_users_to_skip_csv=cassandra
+    ```
+
+    {{< note title="Note" >}}
+In the above sample configuration, we are using an [online LDAP test server](https://www.forumsys.com/tutorials/integration-how-to/ldap/online-ldap-test-server/) for setting up the LDAP authentication with YugabyteDB.
+    {{< /note >}}
+
+    The --ycql_ldap_users_to_skip_csv=cassandra gflag allows access to the user `cassandra` with password authentication. This allows the administrator to log in for setting up the roles (and permissions) for the LDAP users.
+
+1. Start the YugabyteDB cluster.
+
+1. Open the YSQL shell (ysqlsh), specifying the `cassandra` user and prompting for the password.
+
+    ```sh
+    $ ./ycqlsh -u cassandra
+    ```
+
+    When prompted for the password, enter `cassandra` (default password of `cassandra` user). You should be able to login and see a response like below.
+
+    ```output
+    Connected to local cluster at 127.0.0.1:9042.
+    [ycqlsh 5.0.1 | Cassandra 3.9-SNAPSHOT | CQL spec 3.4.2 | Native protocol v4]
+    Use HELP for help.
+    cassnadra@ycqlsh>
+    ```
+
+1. Configure database role(s) for the LDAP user(s).
+
+    We are creating a `ROLE` for username riemann supported by the test LDAP server.
+
+    ```sql
+    cassandra@ycqlsh> create role adam with login=true;
+    ```
+
+1. Connect using LDAP authentication.
+
+    Connect ysqlsh using the `riemann` LDAP user and password specified in the [Online LDAP Test Server](https://www.forumsys.com/tutorials/integration-how-to/ldap/online-ldap-test-server/) page. 
+
+    ```sh
+    $ ./ycqlsh -u riemann
+    ```
```

Comment:
nit: might be nice to have the output/shell after this command works. 
E.g. like the example above but with the new username (or w/e ycqlsh actually outputs):
```
Connected to local cluster at 127.0.0.1:9042.
[ycqlsh 5.0.1 | Cassandra 3.9-SNAPSHOT | CQL spec 3.4.2 | Native protocol v4]
Use HELP for help.
riemann@ycqlsh>
```

Reply:
done